planner/core/cbo_test.go

```diff
@@ -279,6 +279,10 @@ func (s *testAnalyzeSuite) TestIndexRead(c *C) {
 		testKit.MustExec(fmt.Sprintf("insert into t1 values(%v, %v)", i, i))
 	}
 	testKit.MustExec("analyze table t1")
+	ctx := testKit.Se.(sessionctx.Context)
+	sessionVars := ctx.GetSessionVars()
+	sessionVars.HashAggFinalConcurrency = 1
```

Comment:
Because previously when we were writing these test cases, the old cost model does not take concurrency into consideration, so stream aggregation is preferred when child input is in order. But  the new cost model would prefer hash aggregation using the default concurrency variables, which would make a lot of existing test cases meaningless, so I set the concurrency to be 1 to alleviate its effect.